src/jit/simd.cpp

```diff
@@ -2617,6 +2621,61 @@ GenTreePtr Compiler::impSIMDIntrinsic(OPCODE                opcode,
         }
         break;
 
+        case SIMDIntrinsicConvertToInt64:
+        case SIMDIntrinsicConvertToUInt64:
+        {
+#ifdef _TARGET_AMD64_
+            op1 = impSIMDPopStack(simdType, instMethod);
+
+            simdTree = gtNewSIMDNode(simdType, op1, nullptr, simdIntrinsicID, baseType, size);
+            retVal   = simdTree;
+#else
+            JITDUMP("SIMD Conversion is not supported on this platform\n");
+            return nullptr;
```

Comment:
Can you be more specific here, since ConvertToSingle/Double/Int32/UInt32 is supported. E.g.,
```
JITDUMP("SIMD Conversion to Int64/UInt64 is not supported on this platform\n");
```